types/postcss-less/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for postcss-less 3.1
+// Project: https://github.com/shellscape/postcss-less
+// Definitions by: Masafumi Koba <https://github.com/ybiquitous>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as postcss from 'postcss';
+
+export = postcssLess;
+
+declare const postcssLess: postcss.Syntax & {
+    parse: postcss.Parser;
+    stringify: postcss.Stringifier;
+};
```

Comment:
Does this need `nodeToString` ?

Reply:
@sheetalkamat Thanks for the review! I've added it and more test cases via 9e2b40d4b3. 👍